anoncreds/src/services/issuer.rs

```diff
@@ -219,61 +218,60 @@ pub fn create_revocation_status_list(
     )
 }
 
+/// Update the timestamp only without changing any actual state
+pub fn update_revocation_status_list_timestamp_only(
```

Comment:
This function is not used anywhere right? Maybe we should add some tests for it?

Reply:
you are right, I will update the issue of the Todos for revocation status list